geode-core/src/main/java/org/apache/geode/management/MemberMXBean.java

```diff
@@ -877,6 +877,12 @@ String processCommand(String commandString, Map<String, String> env,
    */
   int getOffHeapFragmentation();
 
+  long getOffHeapFragments();
+
+  long getOffHeapFreedChunks();
+
+  int getOffHeapLargestFragment();
+
```

Comment:
Should these be `default method` to avoid the API compatibility problem? Might fix the issue with japicmp, too.

If not applicable or appropriate, please explain and I can remove the change request.

Reply:
I tried having them with a default implementation (returning 0) but it did not fix the issue with japicmp.

According to the discussion on the mailing list about this topic (https://lists.apache.org/thread/jl4ykmyv7nzs5xd9q6vkmvj78vt4jwm5) I consider it reasonable and harmless to add these new methods to the public interface.

Reply:
Sounds good.

Reply:
@dschneider-pivotal @kirklund Could you please review this PR?

Thanks